controller/test-files/work_item_type/list/ok.res.payload.golden.json

```diff
@@ -71,6 +71,14 @@
               "kind": "list"
             }
           },
+          "system.number": {
+            "description": "The unique number that was given to this workitem within its space.",
+            "label": "Number",
+            "required": false,
```

Comment:
Why is the `required` field set to false? Since every workitem belongs to a space, it should have a corresponding system number.

Reply:
Required means, that you have to pass this field upon creation.